remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java

```diff
@@ -141,8 +151,6 @@ public void testDeleteTagById_nonExistentTag()
 
         then(nodesMock).should().validateNode(StoreRef.STORE_REF_WORKSPACE_SPACESSTORE, "dummy-id");
         then(nodesMock).shouldHaveNoMoreInteractions();
-
-        then(taggingServiceMock).shouldHaveNoInteractions();
```

Comment:
This seems like a sensible assertion for a test involving a non-existent tag. Why did we remove it? What method in `taggingService` gets called now?

Reply:
Yes it was by mistake, we reverted the changes.